canonical-websites/www.ubuntu.com
# update /internet-of-things/digital-signage to vbt

## Done

* update /internet-of-things/digital-signage to vbt
* updated heading to header on the overview too

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/internet-of-things/digital-signage)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/7.%20IoT?preview=iot-digital-signage.png) and the [demo](http://www.ubuntu.com-1593-internet-of-things-digital-signage.demo.haus/internet-of-things/digital-signage)

## Issue / Card

Fixes #1593
